app.js

```diff
@@ -9,6 +9,14 @@ app.get("/api/topics", getTopics)
 
 app.get("/api", getApiEndpoints)
 
+app.get("/api/articles/:article_id", getArticleById)
+
+
+
+app.use(( req, res) => {
```

Comment:
this needs to use err and next and not just send back a 404. If you specify in your Promise.reject a status code and an error message, you can use this middleware to just send back that status code and message. This will work for other errors then, not just 404 not found